live-examples/html-examples/forms/css/button.css

```diff
@@ -0,0 +1,26 @@
+.styled {
+  border: 0;
+  line-height: 2.5;
+  padding: 0 20px;
+  font-size: 1rem;
+  text-align: center;
+  color: white;
+  text-shadow: 1px 1px 1px black;
+  border-radius: 10px;
+  background-color: rgba(220,0,0,1);
+  background-image: linear-gradient(to top left,
+                                    rgba(0,0,0,0.2),
+                                    rgba(0,0,0,0.2) 30%,
+                                    rgba(0,0,0,0));
+  box-shadow: inset 2px 2px 3px rgba(255,255,255,0.6),
+              inset -2px -2px 3px rgba(0,0,0,0.6);
```

Comment:
Nit: Code style: space after commas. Also, for number fractions, leave of the 0 i.e.

```
inset -2px -2px 3px rgba(0, 0, 0, .6);
```